src/test/java/org/broadinstitute/hellbender/utils/config/ConfigMarbleRollIntegrationTest.java

```diff
@@ -0,0 +1,56 @@
+package org.broadinstitute.hellbender.utils.config;
+
+import htsjdk.samtools.ValidationStringency;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.GATKBaseTest;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.tools.PrintReads;
+import org.broadinstitute.hellbender.utils.test.ArgumentsBuilder;
+import org.broadinstitute.hellbender.utils.test.IntegrationTestSpec;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.util.Collections;
+
+/**
+ * Integration test suite for Configuration tests.
+ * Created by jonn on 10/4/17.
+ */
+public class ConfigMarbleRollIntegrationTest extends CommandLineProgramTest  {
```

Comment:
Give this class a less confusing name (perhaps `ConfigPrintReadsIntegrationTest`)

Reply:
Sure.  I am used to calling any first-pass simple test (i.e. just to test the flow/machinery) a `marble roll`, so I thought it was straight-forward.